core/src/daemon_utils.rs

```diff
@@ -21,6 +23,25 @@ pub fn lock_daemon_file() -> Result<File, RemuxLibError> {
     Ok(file)
 }
 
+pub fn get_sock_path() -> Result<PathBuf, String> {
```

Comment:
I think string as error is discourages - we should have a RemuxLibError variant I think